examples/mfem/tap.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
Can we use the `tap.sh` from above instead of copying it here? A copy means people will need to maintain both, but they never do, thus bugs hit more than once.